src/templates/partials/header.mjml

```diff
@@ -1,16 +1,16 @@
 <mj-section padding="0px 0px 15px 0px">
-      <mj-column vertical-align="middle">
-        <mj-text align="left" font-size="15px" padding="0px">
-          <p style="text-align: center; margin: 10px 0px;">
-            <span>Email not displaying correctly? </span>
-            <a style="font-size: 15px;" href="{{ browserLink }}" class="link">View it in your browser</a>
-          </p>
-        </mj-text>
-      </mj-column>
-    </mj-section>
+  <mj-column vertical-align="middle">
+    <mj-text align="left" font-size="15px" padding="0px">
+      <p style="text-align: center; margin: 10px 0px;">
+        <span>Email not displaying correctly? </span>
+        <a style="font-size: 15px;" href="{{ browserLink }}" class="link">View it in your browser</a>
+      </p>
+    </mj-text>
+  </mj-column>
+</mj-section>
 
-    <mj-section background-size="100% 100%" background-url="{{ domain }}/bkgd-pitch.png" padding="25px 200px" border-radius="10px 10px 0px 0px">
-      <mj-column background-color="transparent">
-        <mj-image align="center" alt="Football Image" border="none" height="auto" src="{{ domain }}/football.svg" title="Football Image" width="150px"></mj-image>
-      </mj-column>
-    </mj-section>
+<mj-section background-repeat="no-repeat" background-size="100% 100%" background-url="https://nfl-dev.asitewithnoname.com/bkgd-pitch.png" padding="25px 20px">
```

Comment:
@balexander82 It doesn't really make sense to lock this to dev.  We have domain in there to specifically let it go to each env.  I assume you did this for testing since locally it won't work (since your localhost is not publicly accessible).  If true, we should not commit this change.  If there is another reason to lock this, I would much rather see it locked to prod instead of dev.